src/XdebugHandler.php

```diff
@@ -30,6 +30,7 @@ class XdebugHandler
     private $envAllowXdebug;
     private $envOriginalInis;
     private $loaded;
+    /** @var Status */
```

Comment:
without this, since `$statusWriter` is assigned lazily via the setter IDEs are not able to pick up the right type